webui/src/pages/NodesDashboard/NodesTable/NodesTable.tsx

```diff
@@ -86,5 +86,3 @@ const NodesTable: React.FC<TableProps> = ({ data }) => {
     </div>
   );
 };
-
-export default NodesTable;
```

Comment:
Wondering why all export defaults have been replaced with export?

Reply:
https://medium.com/@islizeqiang/why-you-should-avoid-default-exports-in-javascript-modules-7d087dde7ac6#:~:text=Default%20Exports%20are%20Naming%20Nightmares&text=Because%20they%20can%20be%20imported%20with%20any%20name%2C%20different%20developers,maintain%20as%20a%20codebase%20grows.